src/main/java/org/prebid/server/bidder/yieldmo/YieldmoBidder.java

```diff
@@ -78,9 +86,31 @@ private ExtImpYieldmo parseImpExt(Imp imp) throws PreBidException {
         }
     }
 
-    private Imp modifyImp(Imp imp, ExtImpYieldmo ext) {
+    private Imp modifyImp(Imp imp, BidRequest bidRequest, ExtImpYieldmo ext) {
         final YieldmoImpExt modifiedExt = YieldmoImpExt.of(ext.getPlacementId(), extractGpid(imp));
-        return imp.toBuilder().ext(mapper.mapper().valueToTree(modifiedExt)).build();
+
+        Price bidFloorPrice = Price.of(imp.getBidfloorcur(), imp.getBidfloor());
+        bidFloorPrice = BidderUtil.isValidPrice(bidFloorPrice)
+                ? convertBidFloor(bidFloorPrice, imp.getId(), bidRequest) : bidFloorPrice;
+
+        return imp.toBuilder()
+            .bidfloor(bidFloorPrice.getValue())
+            .bidfloorcur(bidFloorPrice.getCurrency())
+            .ext(mapper.mapper().valueToTree(modifiedExt))
+            .build();
+    }
+
+    private Price convertBidFloor(Price bidFloorPrice, String impId, BidRequest bidRequest) {
```

Comment:
This logic id not covered with unit tests

Reply:
Tests have now been added, sorry about that.